src/details/ArborX_DetailsConcepts.hpp

```diff
@@ -60,6 +61,91 @@ struct has_centroid<
 {
 };
 
+// is_complete implementation taken from https://stackoverflow.com/a/44229779
+template <class T, std::size_t = sizeof(T)>
+std::true_type is_complete_impl(T *);
+
+std::false_type is_complete_impl(...);
```

Comment:
is the `...` just a place holder for any type? C++14?

Reply:
It works in C

Reply:
This is one of the usual TMP tricks. The ellipsis is always is the worst candidate in overload resolution. Due to SFINAE, the second overload is only selected if `sizeof(T)` is invalid.

Reply:
makes sense